### PR TITLE
Cosmetic fixes for the systemD Env file

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6306

### CODE REVIEW

File: api/imapsync/update [hunk 1]
```diff
@@ -77,15 +77,15 @@ if ($cmd eq 'update') {
     my $exclude = join ('|',@{$input->{'Exclude'}}) || '';
     $exclude = '|'.$exclude  if ($exclude ne '');
     # write the Env file for systemd
-    my $vars = qq(
-    LOCALUSER=$input->{'name'}
+    my $vars = qq(    LOCALUSER=$input->{'name'}
```
Comment (DavidePrincipi) [inline api/imapsync/update:80]: Whey the leading blanks?

```suggestion
    my $vars = qq(LOCALUSER=$input->{'name'}
```

Comment (gsanchietti) [reply api/imapsync/update:80]: With this fix, the file still has spaces at the begging of each line: it's just a consequence of code indention.

File: api/imapsync/update [hunk 2]
```diff
@@ -77,15 +77,15 @@ if ($cmd eq 'update') {
...
     REMOTEHOSTNAME=$input->{'hostname'}
     REMOTEPORT=$input->{'Port'}
     SECURITY=$crypt
     DELETE=$delete
     DELETEFOLDERS=$deletefolders
     EXCLUDE=$exclude
```
Comment (gsanchietti) [inline api/imapsync/update:86]:
```suggestion
    REMOTEHOSTNAME=$input->{'hostname'}
REMOTEPORT=$input->{'Port'}
SECURITY=$crypt
DELETE=$delete
DELETEFOLDERS=$deletefolders
EXCLUDE=$exclude
```

File: api/imapsync/update [hunk 3]
```diff
@@ -77,15 +77,15 @@ if ($cmd eq 'update') {
...
-    DELETE=$delete
-    DELETEFOLDERS=$deletefolders
-    EXCLUDE=$exclude
-    REMOTEUSERNAME=$input->{'username'});
+REMOTEPORT=$input->{'Port'}
+SECURITY=$crypt
+DELETE=$delete
+DELETEFOLDERS=$deletefolders
+EXCLUDE=$exclude
+    REMOTEUSERNAME=$input->{'username'}
```
Comment (gsanchietti) [inline api/imapsync/update:87]:
```suggestion
REMOTEUSERNAME=$input->{'username'}
```

File: api/imapsync/update [hunk 4]
```diff
@@ -77,15 +77,15 @@ if ($cmd eq 'update') {
     my $exclude = join ('|',@{$input->{'Exclude'}}) || '';
     $exclude = '|'.$exclude  if ($exclude ne '');
     # write the Env file for systemd
-    my $vars = qq(
-    LOCALUSER=$input->{'name'}
+    my $vars = qq(LOCALUSER=$input->{'name'}
     REMOTEHOSTNAME=$input->{'hostname'}
```
Comment (gsanchietti) [inline api/imapsync/update:81]:
```suggestion
REMOTEHOSTNAME=$input->{'hostname'}
```

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**